facilitator/src/bin/facilitator.rs

```diff
@@ -455,8 +455,8 @@ fn main() -> Result<(), anyhow::Error> {
                         .required(true),
                 )
                 .arg(
-                    Arg::with_name("facilitator-ecies-private-key")
-                        .long("facilitator-ecies-private-key")
+                    Arg::with_name("facilitator-ecies-public-key")
+                        .long("facilitator-ecies-public-key")
                         .env("FACILITATOR_ECIES_PRIVATE_KEY")
```

Comment:
The environment variable names should match the parameter names.

Reply:
Oh whoops